Running in wrapper mode for '20005'

Running python job.

[name: "CPU-QUOTA"
int_val: 78
int_val: 53
int_val: 65
int_val: 38
int_val: 40
int_val: 69
int_val: 31
, name: "DISK"
int_val: 17
int_val: 48
int_val: 57
int_val: 26
int_val: 49
int_val: 21
int_val: 52
, name: "MEMORY"
int_val: 31
int_val: 21
int_val: 50
int_val: 54
int_val: 49
int_val: 40
int_val: 53
, name: "NET"
int_val: 35
int_val: 28
int_val: 25
int_val: 33
int_val: 14
int_val: 52
int_val: 49
]
The paramters are {u'CPU-QUOTA': array([78, 53, 65, 38, 40, 69, 31]), u'NET': array([35, 28, 25, 33, 14, 52, 49]), u'DISK': array([17, 48, 57, 26, 49, 21, 52]), u'MEMORY': array([31, 21, 50, 54, 49, 40, 53])}
{'13.57.236.233': ['postgres', 'kibana'], '52.53.237.168': ['node-apt-app', 'kibana'], '13.57.232.190': ['elasticsearch', 'mysql'], '13.56.13.241': ['node-apt-app', 'logstash'], '13.57.58.49': ['haproxy', 'kibana'], '54.153.85.122': ['node-apt-app', 'kibana']}
Setting cpu quota to 77500 out of 250000
STRESSING VM_IP 13.57.236.233 AND CONTAINER fd4563743d87, NET 250880.0
STRESSING VM_IP 13.57.236.233 AND CONTAINER fd4563743d87, DISK 43620761.6
STRESSING VM_IP 13.57.236.233 AND CONTAINER fd4563743d87, MEMORY 1987500000.0
Recovering the memory utilization
Setting cpu quota to 132500 out of 250000
STRESSING VM_IP 13.57.236.233 AND CONTAINER 6cb395f9a474, NET 143360.0
STRESSING VM_IP 13.57.236.233 AND CONTAINER 6cb395f9a474, DISK 40265318.4
STRESSING VM_IP 13.57.236.233 AND CONTAINER 6cb395f9a474, MEMORY 787500000.0
Recovering the memory utilization
Setting cpu quota to 195000 out of 250000
STRESSING VM_IP 52.53.237.168 AND CONTAINER 72bf70afc99a, NET 179200.0
STRESSING VM_IP 52.53.237.168 AND CONTAINER 72bf70afc99a, DISK 14260633.6
STRESSING VM_IP 52.53.237.168 AND CONTAINER 72bf70afc99a, MEMORY 1162500000.0
Recovering the memory utilization
Setting cpu quota to 132500 out of 250000
STRESSING VM_IP 52.53.237.168 AND CONTAINER 6a93346d70ee, NET 143360.0
STRESSING VM_IP 52.53.237.168 AND CONTAINER 6a93346d70ee, DISK 40265318.4
STRESSING VM_IP 52.53.237.168 AND CONTAINER 6a93346d70ee, MEMORY 787500000.0
Recovering the memory utilization
Setting cpu quota to 162500 out of 250000
STRESSING VM_IP 13.57.232.190 AND CONTAINER f914a2e03362, NET 128000.0
STRESSING VM_IP 13.57.232.190 AND CONTAINER f914a2e03362, DISK 47815065.6
STRESSING VM_IP 13.57.232.190 AND CONTAINER f914a2e03362, MEMORY 1875000000.0
Recovering the memory utilization
Setting cpu quota to 100000 out of 250000
STRESSING VM_IP 13.57.232.190 AND CONTAINER c6bd985fa324, NET 71680.0
STRESSING VM_IP 13.57.232.190 AND CONTAINER c6bd985fa324, DISK 41104179.2
STRESSING VM_IP 13.57.232.190 AND CONTAINER c6bd985fa324, MEMORY 1837500000.0
Recovering the memory utilization
Setting cpu quota to 195000 out of 250000
STRESSING VM_IP 13.56.13.241 AND CONTAINER f8b10d691b77, NET 179200.0
STRESSING VM_IP 13.56.13.241 AND CONTAINER f8b10d691b77, DISK 14260633.6
STRESSING VM_IP 13.56.13.241 AND CONTAINER f8b10d691b77, MEMORY 1162500000.0
Recovering the memory utilization
Setting cpu quota to 95000 out of 250000
STRESSING VM_IP 13.56.13.241 AND CONTAINER 197dde384659, NET 168960.0
STRESSING VM_IP 13.56.13.241 AND CONTAINER 197dde384659, DISK 21810380.8
STRESSING VM_IP 13.56.13.241 AND CONTAINER 197dde384659, MEMORY 2025000000.0
Recovering the memory utilization
Setting cpu quota to 172500 out of 250000
STRESSING VM_IP 13.57.58.49 AND CONTAINER ce162cfce35f, NET 266240.0
STRESSING VM_IP 13.57.58.49 AND CONTAINER ce162cfce35f, DISK 17616076.8
STRESSING VM_IP 13.57.58.49 AND CONTAINER ce162cfce35f, MEMORY 1500000000.0
Recovering the memory utilization
Setting cpu quota to 132500 out of 250000
STRESSING VM_IP 13.57.58.49 AND CONTAINER ecf769a2a794, NET 143360.0
STRESSING VM_IP 13.57.58.49 AND CONTAINER ecf769a2a794, DISK 40265318.4
STRESSING VM_IP 13.57.58.49 AND CONTAINER ecf769a2a794, MEMORY 787500000.0
Recovering the memory utilization
Setting cpu quota to 195000 out of 250000
STRESSING VM_IP 54.153.85.122 AND CONTAINER f340b9f2855e, NET 179200.0
STRESSING VM_IP 54.153.85.122 AND CONTAINER f340b9f2855e, DISK 14260633.6
STRESSING VM_IP 54.153.85.122 AND CONTAINER f340b9f2855e, MEMORY 1162500000.0
Recovering the memory utilization
Setting cpu quota to 132500 out of 250000
STRESSING VM_IP 54.153.85.122 AND CONTAINER 93ffc048982e, NET 143360.0
STRESSING VM_IP 54.153.85.122 AND CONTAINER 93ffc048982e, DISK 40265318.4
STRESSING VM_IP 54.153.85.122 AND CONTAINER 93ffc048982e, MEMORY 787500000.0
Recovering the memory utilization
Using 1 trials
Initializing Machines
ab -p post.json -T application/json -n 800 -c 500 -s 9999 -e results_file http://13.57.58.49:80/app/psql/users/
ab -p post.json -T application/json -n 800 -c 500 -s 9999 -e results_file http://13.57.58.49:80/app/mysql/users/
Sleeping for 2 seconds
ab -q -n 200 -c 50 -s 9999 -e results_file http://13.57.58.49:80/app/psql/users/ > output0.txt
ab -q -p post.json -T application/json -n 200 -c 50 -s 9999 -e results_file http://13.57.58.49:80/app/psql/users/ > output1.txt
ab -q -n 800 -c 500 -s 9999 -e results_file http://13.57.58.49:80/app/mysql/users/ > output2.txt
ab -q -p post.json -T application/json -n 800 -c 500 -s 9999 -e results_file http://13.57.58.49:80/app/mysql/users/ > output3.txt
ab -q -n 800 -c 500 -s 9999 -e results_file http://13.57.58.49:80/app/users/ > output4.txt
ab -n 1 -s 9999 -e results_file http://13.57.58.49:80/app/elastic/users/3 > output5.txt
Please ignore the following new lines (if any)
32.97,1516.606,537.0,4200.0,4651.0
35.8,1396.471,587.0,4100.0,4329.0
58.79,8504.866,4298.0,10257.0,11347.0
62.55,7993.847,4453.0,10697.0,10774.0
174.28,2868.934,70.0,91.0,260.0
0.23,4324.739,-1,-1,-1
total:364.62,26605.463,11372.16387,30772.16387,32788.16387
Sleeping for 15 seconds for proper deletion
Sleeping for 5 more seconds for proper deletion
Experiment results are {'latency_90': [30772.16387], 'latency': [26605.463000000003], 'rps': [364.62], 'latency_50': [11372.16387], 'latency_99': [32788.16387]}
Got result 32788.163870

----------------------------------------
Problem running the job:
(<class 'subprocess.CalledProcessError'>, CalledProcessError(), <traceback object at 0x114ec3908>)
Traceback (most recent call last):
  File "/Users/rahulbalakrishnan/Desktop/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/Users/rahulbalakrishnan/Desktop/throttlebot/src/spearmint/spearmint/runner.py", line 148, in run_python_job
    save_job(job)
  File "/Users/rahulbalakrishnan/Desktop/throttlebot/src/spearmint/spearmint/helpers.py", line 72, in save_job
    file_write_safe(filename, job.SerializeToString())
  File "/Users/rahulbalakrishnan/Desktop/throttlebot/src/spearmint/spearmint/helpers.py", line 46, in file_write_safe
    sh(cmd)
  File "/Users/rahulbalakrishnan/Desktop/throttlebot/src/spearmint/spearmint/helpers.py", line 19, in sh
    subprocess.check_call(cmd, shell=True)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 540, in check_call
    raise CalledProcessError(retcode, cmd)
CalledProcessError: Command 'mv "/var/folders/0x/f3sy_kt92pbfpm7_jg_d1gqm0000gn/T/tmpAX60rl" "/Users/rahulbalakrishnan/Desktop/throttlebot/src/spearmint/bayOptSearch/jobs/00020005.pb"' returned non-zero exit status 1
None
----------------------------------------
Process Process-131:
Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/rahulbalakrishnan/Desktop/throttlebot/src/spearmint/spearmint/runner.py", line 62, in job_runner
    job      = load_job(job_file)
  File "/Users/rahulbalakrishnan/Desktop/throttlebot/src/spearmint/spearmint/helpers.py", line 76, in load_job
    fh = open(filename, 'rb')
IOError: [Errno 2] No such file or directory: u'/Users/rahulbalakrishnan/Desktop/throttlebot/src/spearmint/bayOptSearch/jobs/00020005.pb'
